Makefile

```diff
@@ -197,6 +197,17 @@ kind-node-image:
 kubectl-version:
 	@echo $(KUBECTL_VERSION)
 
+kind-test-packages:
+	@./control-plane/build-support/scripts/set_test_package_matrix.sh "acceptance/ci-inputs/kind_acceptance_test_packages.yaml"
```

Comment:
I had to go against putting this in a variable at the top of the file like the others because of the json output. When you put it in the variable at the top it does weird things with the `""` making it invalid JSON